### PR TITLE
test(statuscake): add test to my regex

### PR DESCRIPTION
**Pull request by VirgileLyon**
In this PR : 

- Test my regex / filter 🔭 
- Update to my doc 📝

### CODE REVIEW

File: src/___tests__/statuscake.test.js [hunk 1]
```diff
@@ -15,6 +16,9 @@ jest.mock('../statuscakedata', () => ({
   setData: jest.fn(),
 }));
 
+beforeEach(() => {
+  storage.setData = jest.fn();
```
Comment (hpierre74) [inline src/___tests__/statuscake.test.js]: use `jest.clearAllMocks();` instead, as your mock already exist :)

File: src/___tests__/statuscake.test.js [hunk 2]
```diff
@@ -45,6 +49,46 @@ test('fetchStatuCakeData call axios', async () => {
   expect(storage.setData).toBeCalledTimes(1);
 });
 
+test('it should filter metrics base on config', async () => {
+  axios.get.mockResolvedValue({
+    data: {
```
Comment (hpierre74) [inline src/___tests__/statuscake.test.js]: it looks like your data is duplicated, you can put it elswhere as a constant, or even make a util function to build it in different manners

File: src/___tests__/statuscake.test.js (hunk 1 above)
Comment (VirgileLyon) [reply src/___tests__/statuscake.test.js]: 👀

File: src/___tests__/statuscake.test.js (hunk 2 above)
Comment (VirgileLyon) [reply src/___tests__/statuscake.test.js]: Exact, done 🐶

File: README.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ To configure the project, we use npm [`config`](https://www.npmjs.com/package/co
 - nameOfFileSizeGauge (Name of your filesize gauge)
 - nameOfRequestGauge (Name of your request gauge)
 - tagsFromTestTitleRegexp (Your regex)
+- tags: (Your tags)
```
Comment (hpierre74) [inline README.md]: rm the ":" to be consistent 😛

Comment (VirgileLyon) [reply README.md]: Thanks.

File: src/___tests__/statuscake.test.js [hunk 4]
```diff
@@ -15,36 +16,64 @@ jest.mock('../statuscakedata', () => ({
...
+test('it should filter metrics base on config', async () => {
+  axios.get.mockResolvedValue(
+    block,
+  );
+  config.statuscake.testTitleFilter = 'NCIS';
+
+  await fetchStatuCakeData(storage);
+
+  expect(storage.setData).toBeCalledTimes(1);
+  expect(storage.setData).toBeCalledWith([{
```
Comment (hpierre74) [inline src/___tests__/statuscake.test.js]: you could have retrieve it directly from your block constant instead of rewriting it 💡

**Pull request merged**